app/main.py

```diff
@@ -2,5 +2,17 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    funcs = {}
+
+    def wrapper(*args, **kwargs) -> Callable:
```

Comment:
The wrapped function does not always return `Callable`, replace the return type annotation with `Any` to have an actual representation.